### ISSUE TITLE
update repo to match specs

### ISSUE THREAD

**Issue by jesusff** [WCRP-CORDEX/archive-specifications#6]
Hi,

now that the [final archiving specs](https://zenodo.org/doi/10.5281/zenodo.10961068) are out and the repositories under the CORDEX GitHub organization could be explored more thoroughly in search for authoritative information, we should consider updating the contents of this "archive-specifications" repository to match the actual specs.

I assume this repo was initially thought to have the specs nicely displayed as a JupyterBook (e.g. https://wcrp-cordex.github.io/archive-specifications/attributes.html), but the information is currently not up to date. This can be quite confusing for people landing in this repository.

As a quick fix, we could replace the contents of the default branch in this repo by a README pointing to the actual archiving specs, while we (decide whether to) update the JupyterBook sources.

**Comment by larsbuntemeyer** [comment]
Yes, good point. I'll have to fix this (or probably take it offline) to avoid conflicts! I have som time on (innovation) friday to go through some CORDEX issues...

**Comment by jesusff** [comment]
For the moment, I have deactivated the GitHub Pages in this repo and removed the jupyterbook sources from the main branch. They are under the [jupyterbook](https://github.com/WCRP-CORDEX/archive-specifications/tree/jupyterbook) branch. I leave this issue open as a reminder that this work is here, ready to have future specs maintained as traceable markdown sources.

I close other related issues (#3, #8) related to the Jupyterbook, even though they will need to be addressed at some point when this work is retaken.

**Comment by larsbuntemeyer** [comment]
Thanks for cleaning this!

**Comment by jesusff** [comment]
The mkdocs version is now in place and matching the official specs in Zenodo
